src/javascript/expression.rs

```diff
@@ -201,7 +230,11 @@ impl<'module> Generator<'module> {
         for (i, expression) in expressions.iter().enumerate() {
             if i + 1 < count {
                 documents.push(self.not_in_tail_position(|gen| gen.expression(expression))?);
-                documents.push(";".to_doc());
+                match expression {
+                    // Assignment generates multiple lines of JS and so handles it's own ;
```

Comment:
I think the assignment shouldn't generate a trailing semicolon, as with other expressions

Reply:
Actually I'm not sure that would be possible